modules/concertBidAdapter.js

```diff
@@ -42,7 +42,7 @@ export const spec = {
         pageUrl: bidderRequest.refererInfo.page,
         screen: [window.screen.width, window.screen.height].join('x'),
         debug: debugTurnedOn(),
-        uid: getUid(bidderRequest),
+        uid: getUid(bidderRequest, validBidRequests),
         optedOut: hasOptedOutOfPersonalization(),
         adapterVersion: '1.1.1',
```

Comment:
Can we bump the adapter version? This is useful in case want to do something based on the adapter version on the CBS side.